lib/gingr/import_util.rb

```diff
@@ -32,14 +32,13 @@ def index_solr_from_dir(directory_path, url, reference_urls)
       end
 
       def get_reference_urls(options)
-        update_reference_field = options[:update_reference_field]
-        return {} unless update_reference_field
-
-        hash = {}
-        Config.reference_urls.each_key do |key|
-          hash[key] = reference_url(key, options)
+        {}.tap do |refs|
```

Comment:
The logic is the same, this just uses the `tap` idiom to avoid having to return `hash` at the end.